regulations/static/regulations/js/source/views/comment-review-view.js

```diff
@@ -56,6 +60,16 @@ var CommentReviewView = Backbone.View.extend({
     });
     if (target) {
       target.remove();
+
+      this.numberOfComments--;
+      this.checkForComments();
+    }
+  },
+
+  checkForComments: function() {
+    if (this.numberOfComments === 0) {
```

Comment:
Great point, I wasn't thorough enough because I wondered why `this.comments` did not change after a comment was removed and decided to make a new variable instead of investigating the event. Now I updated so `this.comments` will update after removal.